source/js/components/join/join.jsx

```diff
@@ -449,11 +495,14 @@ export default class JoinUs extends React.Component {
                 ref={el => (this.privacy = el)}
                 required
               />
-              <p className="d-inline-block body-small form-text mb-0">
-                {getText(
-                  `I'm okay with Mozilla handling my info as explained in this Privacy Notice`
-                )}
-              </p>
+              <Localized
+                id="privacy-policy"
+                pp={<a href="https://www.mozilla.org/privacy/websites/"></a>}
```

Comment:
what is `pp`, is this an attribute we get to name, or something baked in? (if the former, let's not call this `pp` but use something indicative of what it is)

Reply:
This is a feature of Fluent: see https://github.com/projectfluent/fluent.js/wiki/React-Overlays.

Reply:
renamed this to `privacyLink` for better clarity